packages/types/src/electra/sszTypes.ts

```diff
@@ -256,13 +249,13 @@ export const SignedBuilderBid = new ContainerType(
   {typeName: "SignedBuilderBid", jsonCase: "eth2"}
 );
 
-export const ExecutionPayloadAndBlobsBundle = new ContainerType(
-  {
-    executionPayload: ExecutionPayload, // Modified in ELECTRA
-    blobsBundle: denebSsz.BlobsBundle,
-  },
-  {typeName: "ExecutionPayloadAndBlobsBundle", jsonCase: "eth2"}
-);
+// export const ExecutionPayloadAndBlobsBundle = new ContainerType(
```

Comment:
`execution_requests` are part of bundle now, see [here](https://github.com/ethereum/builder-specs/pull/101/files#diff-2be1000fe5e3e5a9826d0c753b4acbaa496eb07148ef9b5698b8e3bc660fef89R21)

Reply:
Thanks for the heads up. Will update the builder flow in a separate PR.